ortools/linear_solver/csharp/linear_solver.i

```diff
@@ -302,7 +311,64 @@ CONVERT_VECTOR(operations_research::MPVariable, MPVariable)
 %unignore operations_research::MPSolverParameters::SCALING_OFF;  // no test
 %unignore operations_research::MPSolverParameters::SCALING_ON;  // no test
 
+// Callback API
+
+// Expose callback event type enum
+%unignore operations_research::MPCallbackEvent::kUnknown;
```

Comment:
not sure you need %unignore AND %rename IIRC rename will "also" unignore the value
both are custom macro on based on SWIG command

Reply:
Yes, you are right. Sorry, I got only 3 or 4 days experience with SWIG and C++ diving into or-tools source codes :-) 

Fixed